eng/common/scripts/Verify-Links.ps1

```diff
@@ -201,8 +201,13 @@ function CheckLink ([System.Uri]$linkUri)
   
   if ($checkLinkGuidance) {
     # Check if the url is relative links, suppress the archor link validation.
-    if (!$linkUri.IsAbsoluteUri -and !$linkUri.ToString().StartsWith("#")) {
-      LogWarning "DO NOT use relative link $linkUri. Please use absolute link instead. Check here for more infomation: https://aka.ms/azsdk/guideline/links"
+    if (!$linkUri.IsAbsoluteUri) {
```

Comment:
anchor tags can also be part of an absolute link we might want to figure out which ones we should try and validate.